### PR TITLE
Add support for NVMe format.

### LINKED ISSUE #61: Change fog.wipe to wipe NVMe drives correctly.
The `fog.wipe` script currently uses `shred` or `dd` to wipe out the hard drive.
Wiping an SSD or NVMe drive like this will make the drives deteriorate very quickly.

### PR DESCRIPTION
**Pull request by rluzuriaga**
Resolves #61

### CODE REVIEW

**Comment by lukebarone** [comment]
If you're using the same command line for all the NVMe wipes, why not check if `$isNVMe` is set, and if so, run the wipe... Otherwise, check the other conditions?

**Comment by rluzuriaga** [comment]
@lukebarone I'm not sure where my head was at yesterday when I pushed the code that I couldn't think of how to do it.
I have a nicer looking and non-repetitive solution going out now.

File: Buildroot/board/FOG/FOS/rootfs_overlay/bin/fog.wipe [hunk 1]
```diff
@@ -5,29 +5,38 @@ displayBanner
 disks=""
 disk=""
 hd=""
+isNvme=0
```
Comment (Sebastian-Roth) [inline Buildroot/board/FOG/FOS/rootfs_overlay/bin/fog.wipe:8]: I can see this being even more streamlined by overwriting `wipemode` for that:
```
[[ $hd == *[Nn][Vv][Mm][Ee]* ]] && wipemode="nvme"
...
case $wipemode in
...
    nvme)
        echo -e " Starting disk wipe of $hd using nvme format...\n"
        usleep 10000000
        nvme format $hd --force
...
```

File: Buildroot/board/FOG/FOS/rootfs_overlay/bin/fog.wipe [hunk 2]
```diff
@@ -5,29 +5,38 @@ displayBanner
...
+            usleep 10000000
+            shred -f -v -n 1 "$hd"
+            ;;
+        fast|fastwipe)
+            echo " Writing zeros to start of $hd"
+            dd if=/dev/zero of="$hd" bs=512 count=100000
+            ;;
+    esac
+fi
+echo -e "\n Wiping complete.\n"
```
Comment (Sebastian-Roth) [inline Buildroot/board/FOG/FOS/rootfs_overlay/bin/fog.wipe:41]: Definitely a good idea to move that printout to the end!

**Review by Sebastian-Roth — COMMENTED** [review]
See the comments.

File: Buildroot/board/FOG/FOS/rootfs_overlay/bin/fog.wipe (hunk 1 above)
Comment (rluzuriaga) [reply Buildroot/board/FOG/FOS/rootfs_overlay/bin/fog.wipe:8]: You are right. I was thinking that I would need to also change the wipemode in FOG, but it's not necessary.
I'll add the change.

**Review by Sebastian-Roth — APPROVED** [review]
Nice!

**Pull request merged**